docs/5g-on-helium/5g-on-helium.mdx

```diff
@@ -11,8 +11,59 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 
 # 5G On Helium
 
-:::info Check Back Soon
+Today all mobile networks are built top-down by service providers (e.g., AT&T and Verizon in the
+United States). Future networks will likely be a hybrid of large-scaled operators and people-owned
+5G Hotspots due to demands of consumer behavior (higher bandwidth and lower latency requirements)
+and the physics of wireless radio. More data on the wireless network requires higher frequency
+bands, which requires denser networks with more nodes, which further increases the cost of site
+acquisition.
 
-MOBILE Announced at Consensus. More details coming soon.
+The Helium model decentralizes site acquisition costs and enables communities to participate in
+building reliable, high bandwidth networks using technologies like CBRS and Wi-Fi. This doesn’t
+completely replace service providers altogether. Instead, we see service providers as a critical
+component of this ecosystem.
+
+## CBRS Radios
+
+Citizens Broadband Radio Service (CBRS) is a wide broadcast 3.5Ghz band in the United States that
+does not require a tower operator to acquire spectrum licenses.
+
+More information about CBRS can be found on the
+[Federal Communications Commission Website](https://www.fcc.gov/35-ghz-band-overview)
+
+5G Hotspots can participate in providing 5G cellular coverage when connected to Indoor or Outdoor
+CBRS Small Cell Radios while also participating in providing LoRaWAN coverage.
+
+## Network of Networks
+
+With the passing of
+[HIP 51: Helium DAOs](https://github.com/helium/HIP/blob/main/0051-helium-dao.md) a general
+structure for onboarding new Decentralized Network Protocols (DNPs) to the broader Helium Network
+was created. The Helium Mobile DAO with Helium 5G Hotspots are the first step in the evolution of
+the Helium ecosystem.
+
+## Helium 5G Rollout
+
+The passing of [HIP 51: Helium DAO](https://github.com/helium/HIP/blob/main/0051-helium-dao.md)
+allows for the creation of a Mobile subDAO, which rewards 5G Hotspot operators with MOBILE token on
+the existing Helium blockchain. This will immediately allow any online 5G Hotspot with a Small Cell
+Radio to mine MOBILE tokens. Mobile mining is only available in the US at this time.
```

Comment:
```suggestion
The passing of [HIP 51: Helium DAO](https://github.com/helium/HIP/blob/main/0051-helium-dao.md)
allows for the creation of a MOBILE tokens that will reward 5G Hotspot operators on
the existing Helium blockchain. This will immediately allow any online 5G Hotspot with a Small Cell
Radio to mine MOBILE tokens.
```